src/main/java/ch/hftm/controllers/AlbumController.java

```diff
@@ -275,6 +272,41 @@ public void initializeAlbum() {
 
     }
 
+    @FXML
+    public void editImage() {
+        String title = "";
+        String content = "";
+        if (imageSelection.size() == 1) {
+            Image imageToEdit = imageSelection.get(0);
+            Scene scene = albumPane.getScene();
+
+            SceneController sceneController = new SceneController();
+            try {
+                sceneController.changeScene(scene, "image", imageToEdit);
+            } catch (IOException e) {
+                log.error(String.format("The image %s could not be edited: %s", imageToEdit.getFileName(), e.getMessage()));
+            } finally {
+                imageSelection.clear();
+            }
+        } else if (imageSelection.isEmpty()) {
+            content = "You need to select an image you want to edit.";
+            alertUser(title, content, AlertType.WARNING);
+        } else {
+            content = "Only one image at a time can be edited.";
+            alertUser(title, content, AlertType.WARNING);
+        }
```

Comment:
Again I would propose similar refactoring, if you are interested to know the general principle of simplifying those kind of if/else if/else statements you can read this article [Guard Clause](https://deviq.com/design-patterns/guard-clause)
```suggestion
        if (imageSelection.isEmpty()) {
            content = "You need to select an image you want to edit.";
            alertUser(title, content, AlertType.WARNING);
            return;
        }
        if (imageSelection.size() > 1) {
            content = "Only one image at a time can be edited.";
            alertUser(title, content, AlertType.WARNING);
            return;
        }

        Image imageToEdit = imageSelection.get(0);
        Scene scene = albumPane.getScene();
        SceneController sceneController = new SceneController();
        try {
            sceneController.changeScene(scene, "image", imageToEdit);
        } catch (IOException e) {
            log.error(String.format("The image %s could not be edited: %s", imageToEdit.getFileName(), e.getMessage()));
        } finally {
            imageSelection.clear();
        }
```